repose-aggregator/artifacts/build.gradle

```diff
@@ -0,0 +1,75 @@
+import groovyx.net.http.ContentTypes
+import io.github.httpbuilderng.http.HttpTask
+
+import static groovyx.net.http.NativeHandlers.Encoders
+
+apply plugin: 'io.github.http-builder-ng.http-plugin'
+
+// Informs Gradle that child projects must be evaluated (i.e., configured) before this project.
+// Doing so is necessary in order to use the configurations from child projects to generate
+// the requests for publication (via configuration of the HttpTask).
+evaluationDependsOnChildren()
```

Comment:
I am not a fan of having to use this instruction. Unfortunately, since I opted to use the `http-plugin`, I need to know what artifacts will be pushed at configuration time which means that the child projects to this project must be evaluated before this project.

Of course, this couples this project to its child projects. Which is why I opted to create a new project to contain the package publication task.

A different approach would be to not use the `http-plugin` but to use `http-builder` directly. Then we could define the package publication task ourselves rather than configuring an `HttpTask`. As a result, the package publication task would not need to know which package(s) to publish until execution time. That is, the request generation would occur in the `doLast` closure. Building the requests themselves would even look largely the same, but with a small amount more boilerplate. And of course, switching to a different HTTP client would be more difficult than it is with the plugin (theoretically).